javascript/implementation/IndexedDbStore.ts

```diff
@@ -806,7 +806,7 @@ export class IndexedDbStore implements Store {
             desiredSrc,
             asOfTs
         );
-        const lower = [desiredSrc, Behavior.DIRECTORY];
+        const lower = [desiredSrc];
```

Comment:
Behavior.DIRECTORY was in the spot where the key would be, so anything under the number 4 would not be found. LOL